admin/pre_reg/views.py

```diff
@@ -17,19 +18,18 @@
 )
 from admin.pre_reg import serializers
 from admin.pre_reg.forms import DraftRegistrationForm
+from admin.pre_reg.utils import sort_drafts, SORT_BY
 from framework.exceptions import PermissionsError
 from website.exceptions import NodeStateError
 from website.files.models import FileNode
-from website.project.model import MetaSchema, DraftRegistration
+from website.project.model import DraftRegistration, MetaSchema
 
 from admin.base.utils import PreregAdmin
 
 
 class DraftListView(PreregAdmin, ListView):
     template_name = 'pre_reg/draft_list.html'
-    paginate_by = 10
-    paginate_orphans = 1
-    ordering = '-approval.initiation_date'
+    ordering = '-date'
     context_object_name = 'draft'
 
     def get_queryset(self):
```

Comment:
I know the below code is from a previous PR, but there is a helper function for getting the prereg metaschema: `website.prereg.utils.get_prereg_schema`.